binding.cc

```diff
@@ -1107,6 +1197,77 @@ namespace zmq {
     BufferReference* bufref_;
   };
 
+  NAN_METHOD(Socket::Sendv) {
+    Socket* socket = GetSocket(info);
+    if (socket->state_ != STATE_READY)
+      return info.GetReturnValue().Set(Nan::False());
```

Comment:
stylewise I would suggest using 0 over false

but that means `return info.GetReturnValue().Set(Nan::New<v8::Number>(0));`, 

so maybe the false primitive setter is cleaner